styles.css

```
@@ -507,6 +507,56 @@ body {
 	background-color: white;
 }
 
+.promo-container {
+	padding: 0 0.1em;
```

Замени лучше на
```suggestion
	padding: 0 0.5em;
```
чтобы отступы от краев были такие же как в хедере